awx/main/utils/common.py

```diff
@@ -677,8 +677,14 @@ def get_system_task_capacity(scale=Decimal(1.0)):
     elif settings_forks:
         return int(settings_forks)
 
-    _, cpu_cap = get_cpu_capacity()
-    _, mem_cap = get_mem_capacity()
+    if cpu_capacity is None:
+        _, cpu_cap = get_cpu_capacity()
+    else:
+        cpu_cap = cpu_capacity
+    if mem_capacity is None:
+        _, mem_cap = get_mem_capacity()
+    else:
+        mem_cap = mem_capacity
```

Comment:
This makes a lot of sense how you made these parameters (cpu_capacity and mem_capacity) into optional parameters for the case of a remote machine. I am glad you were able to offload some of the logic (the stuff in this method) onto the management node so that we're not replicating that in the Ansible module.